charts/snowplow-iglu-server/templates/iglu-hooks.yaml

```diff
@@ -118,7 +118,7 @@ metadata:
   labels:
     {{- include "snowplow.labels" $ | nindent 4 }}
   annotations:
-    "helm.sh/hook": post-install,post-upgrade
+    "helm.sh/hook": post-install
```

Comment:
We're doing this for Azure above. Does this require changing as well?

https://github.com/snowplow-devops/helm-charts/blob/4cedf1574d86bb12dc7830b58a4c40a6afad8c09/charts/snowplow-iglu-server/templates/iglu-hooks.yaml#L61

It looks like the same issue would happen where the job would run for any changes and it would attempt to create the database and fail

Reply:
@ianarsenault not necessarily. The difference for Azure is that the hook job does not create a new database, it just creates a new user, I have tested it in Azure and the hook jobs finishes successfully allowing the dev iglu pods to be spawned without issues. We only see the following error in the hook jobs but it does not cause errors in iglu dev pods:

<img width="460" alt="image" src="https://github.com/user-attachments/assets/159f2f4a-8f98-49f0-bc15-4eb7e60e8c5f">

So the first part of the hook job to create a user fails but the rest are successful so this is why the hook job finishes successfully. We can of course remove `post-upgrade` as well cause I can't think of a scenario where it is necessary either